### ISSUE TITLE
The height of the "DropDown" list for the combobox control can't be reset default height after called the " comboBox1.Items.Clear();" method

### ISSUE THREAD

**Issue by Jessie-Zhang01** [dotnet/winforms#339]
**OS** : RS4 X64+.Net 4.8.03316
**VS** : Dev16 d16.0stg 16.0 Preview 3 28505.103

**More info:**
1. This issue can reproduce from VS Dev10 to Dev16.  
2. This issue can't reproduce when adding the following code to Form1.Designer.cs.
```
comboBox1.Items.AddRange(new object[] { "a", "b", "c", "d", "e" });
comboBox1.Items.Clear();
```
**Steps to reproduce:**
1. Create a Winforms application.
2. Drag a Combobox control to form designer.
3. Add the following code to Form1.cs.
 ```
private void Form1_Load(object sender, EventArgs e)
        {
 comboBox1.Items.AddRange(new object[] { "a", "b", "c", "d", "e" });
           comboBox1.Items.Clear();
}
```
**Actual:**
The height of the "DropDown" list for the combobox control can't be reset default height after called the " comboBox1.Items.Clear();" method.
![image](https://user-images.githubusercontent.com/45864649/51449528-6b448380-1d67-11e9-9549-e6e17359c988.png)

**Expected:** 
The height of the "DropDown" list for the combobox control should be reset default height after called the " comboBox1.Items.Clear();" method.
![image](https://user-images.githubusercontent.com/45864649/51449518-61bb1b80-1d67-11e9-8240-5ab5d7b5f8ae.png)

**Comment by OliaG** [comment]
Seems this is not an issue, the Actual result is the desired. If we drag-and-drop empty combobox, we'll get the same Actual result.

**Comment by Jessie-Zhang01** [comment]
@OliaG Sorry, I reversed the actual result and expected result when reporting it. The correct result has been updated in above reproduce steps. Could you please review again and triage it to see if it is a real issue? If yes, please reactive the bug, due to I don't have permission to change it.

**Comment by Jessie-Zhang01** [comment]
@OliaG @merriemcgaw Could you please help to review this issue again if need to reactivate it?

**Comment by Jessie-Zhang01** [comment]
This issue still reproduce on .Net Core 5.0.100-preview.4.20208.26.

**Comment by RussKie** [comment]
Can you please supply a repro project.

**Comment by Jessie-Zhang01** [comment]
Of course, you can use the following project to reproduce this issue:
[ComboBox.zip](https://github.com/dotnet/winforms/files/4455287/ComboBox.zip)

**Comment by RussKie** [comment]
@Jessie-Zhang01 I tried the example in net45 and net472 and it appears to be behaving the same way.... Can you please confirm this is an existing behaviour in .NET Framework?

**Comment by Jessie-Zhang01** [comment]
@RussKie Yes, this issue also reproduce on .Net Framework. And this issue was associated with vs feedback item: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/759614
